src/main/java/it/reply/orchestrator/service/deployment/providers/MarathonServiceImpl.java

```diff
@@ -334,15 +371,43 @@ protected MarathonApp createInternalTaskRepresentation() {
     return new MarathonApp();
   }
 
+
+
   @Override
+  @Deprecated
```

Comment:
Please, do not leave this method now working silently; either: 

- throw an `UnsupportedOperationException`
- or keep the logic functioning without these new parents
- or update the inherited interface and just remove this newly deprecated method